src/core/create_compilers/RollupCompiler.ts

```diff
@@ -97,84 +130,90 @@ export default class RollupCompiler {
 			renderChunk(code: string, chunk: RenderedChunk) {	
 				that.chunks.push(chunk);
 			},
-			transform(code: string, id: string): TransformResult {
-				// rollup-plugin-svelte adds an import statement to the js file which references the css file
-				// that won't be able to be compiled as js, so we remove it here and store a copy to use later
-				if (/\.css$/.test(id)) {
-					that.css_files[id] = code;
-					return {code: '', moduleSideEffects: 'no-treeshake'};
+			renderDynamicImport({ format, moduleId, targetModuleId, customResolution }) {
+				if (targetModuleId) {
+					return {
+						left: 'Promise.all([import(',
+						right: `), /* ___${targetModuleId}___ */]).then(x => x[0])`
```

Comment:
I think I'd be more comfortable if this has a slightly more unique string in the comment.

Reply:
updated this with the fix for terser compatibility